mne/event.py

```diff
@@ -259,10 +259,16 @@ def read_events(filename, include=None, exclude=None, mask=0):
         event_list = lines[:, goods]
         if event_list.shape[0] > 0 and event_list[0, 2] == 0:
             event_list = event_list[1:]
+            warn('first row of event file discarded (zero-valued)')
```

Comment:
@Eric89GXL this warning could probably be improved; since I don't know why this was being done I had to be kind of vague.